Documentation/ApiOverview/FeatureToggles/Index.rst

```diff
@@ -38,11 +38,11 @@ Bad examples:
 
 Good examples:
 
-- `ExtendedRichtextFormat`
-- `NativeYamlParser`
-- `InlinePageTranslations`
-- `TypoScriptParserIncludesAsXml`
-- `NativeDoctrineQueries`
+- `extendedRichtextFormat`
```

Comment:
Can you add the `lowerCamelCase` suggestion to the initial sentence in this section?

Reply:
Done, I hope this is what you expected :-)